daemon/algod/api/algod.oas2.json

```diff
@@ -595,6 +631,106 @@
           "required": true
         }
       ]
+    },
+    "/v2/catchup/{catchpoint}": {
+      "put": {
```

Comment:
post, as discussed?

Reply:
Rotem think it needs to be moved into a gRPC like interface instead of REST. If we're going to do so, I'm going to leave it like that for now.  (since it's going to be deleted anyway )